src/LintingViewExtension/LinterViewModel.cs

```diff
@@ -74,30 +73,21 @@ public List<LinterExtensionDescriptor> AvailableLinters
             get => linterManager.AvailableLinters.ToList();
         }
         
-        /// <summary>
-        /// The default descriptor is used if no linter is needed for the graph. This is a dummy descriptor that has no extension associated to it.
-        /// This property returns the descriptor as a list for binding purposes.
-        /// </summary>
-        public IList<LinterExtensionDescriptor> DefaultDescriptor { get => new List<LinterExtensionDescriptor> { defaultDescriptor }; }
-
         #endregion
 
         public LinterViewModel(LinterManager linterManager, ViewLoadedParams viewLoadedParams)
         {
             this.linterManager = linterManager ?? throw new ArgumentNullException(nameof(linterManager));
             this.viewLoadedParams = viewLoadedParams;
+            this.dispatcher = viewLoadedParams.DynamoWindow.Dispatcher;
```

Comment:
this might be null... I think you should add `?` to all these calls or wrap them in a helper.